src/main/java/codeu/controller/NotificationServlet.java

```diff
@@ -0,0 +1,21 @@
+package codeu.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/*Servlet class responsible for Notification class functions */
```

Comment:
Add tests. For now it will have only one test, you can expand it as you add more features.